### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by gobi004** [stephanrauh/ngx-extended-pdf-viewer#1259]
Hi,The given guide is not working at all. If I try with in stackbliz or local whatever. If its really working then provide the workig stackbliz link not the demo link which is not editable like this https://pdfviewer.net/extended-pdf-viewer/blob.

**Comment by stephanrauh** [comment]
Stackblitz doesn't work with ngx-extended-pdf-viewer. I'm afraid there's no way to solve this problem - it's a Stackblitz architecture issue.

On a local machine, installing the PDF viewer is really simple:

- `ng new my-project``
- `cd my-project`
- `ng add ngx-extended-pdf-viewer`
- Add the new component `app-example-pdf-viewer` to the `app-component.html`

I've updated the instructions yesterday: https://pdfviewer.net/extended-pdf-viewer/getting-started

Does this help you?

**Comment by gobi004** [comment]
Hi,
I'm getting below error while followed your steps.Which i have tried in local.Still your guide is not working not even compiling the code.The error is getting while compling itself.make sure work in stackbliz or any other online tools. which will help us to use the plugin.Trying every time in local is not possible in live app.

Error: ./node_modules/ngx-extended-pdf-viewer/fesm2015/ngx-extended-pdf-viewer.mjs 8600:6-20
Can't import the named export 'ɵɵviewQuery' from non EcmaScript module (only default export is available)

**Comment by stephanrauh** [comment]
Sounds like you're using an old Angular version. If you're starting a new project with Angular 11 or above, you shouldn't see an error message containing Greek letters.

Technically speaking: I've dropped support for the ViewEngine with version 10.5.0. That's because new projects created with Angular 13 or above always use the new Ivy engine. Ivy is more efficient, so I decided to drop pre-Ivy-support roughly two years after Ivy became the default.
